lib/api/admin/admin.ts

```diff
@@ -0,0 +1,17 @@
+import { db} from "../../db";
+
+export function getlockers() {
+    return db.prepare("SELECT * FROM lockers").get()
```

Comment:
This needs to call `.all()` instead of `.get()`.

Reply:
I'd type this out.
```ts
type Locker = {
   id: number;
}

export function getLockers(): Locker[] {
  return db.prepare("SELECT * FROM lockers").all();
}

```